shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -31,7 +32,20 @@
 
 using namespace shell;
 
-@implementation FlutterPlatformPlugin
+@implementation FlutterPlatformPlugin {
+  fml::WeakPtr<UIViewController> _vc;
```

Comment:
Nit: We prefer not to abbreviate instance variable names in code. `_viewController` works better IMO.